src/commands/tick-register.js

```diff
@@ -30,9 +29,8 @@ function register(yargs) {
     if (err)
       throw err
     
-
     server.register(user)
-    .then(user => setConfig('remote', user.couch.url))
+    .then(user => account.setUser(user))
```

Comment:
Should we provide feedback to the user that their user id was added to N documents?